code/include/swoc/Lexicon.h

```diff
@@ -813,4 +839,30 @@ bwformat(BufferWriter &w, bwf::Spec const &spec, Lexicon<E> const &lex) {
   return w;
 }
 
-}} // namespace swoc::SWOC_VERSION_NS
+}} // namespace swoc
+
+namespace std {
+
+template <size_t IDX, typename E> class tuple_element { static_assert("swoc::Lexicon::Pair tuple index out of range"); };
```

Comment:
Clang, sigh. It's annoying that more modern compilers seem to have more problems. They're supposed to get better.

This is odd, though - the ambiguity is `std::tuple_element` vs. `std::__1::tuple_element`. Is this including two different version of the standard library? Or because libswoc is the one that created namespace `std` and it's normally an alias for `std::__1`?